test/CMakeLists.txt

```diff
@@ -771,122 +810,137 @@ if(iodaconv_bufr_ENABLED)
 
   ecbuild_add_test( TARGET  test_iodaconv_bufr_hrs2ioda
                     TYPE    SCRIPT
-                    COMMAND ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
-                    ARGS    netcdf
+                    COMMAND bash
+                    ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                            netcdf
```

Comment:
Why is test_iodaconv_bufr_mhs2ioda (line 805) different from others?

Reply:
@jamiebresch thanks for catching this! It's a typo and I'll fix it.

Reply:
Done!